src/components/common/SearchBox/SearchBox.tsx

```diff
@@ -56,7 +54,6 @@ export const SearchBox: React.FC<SearchBoxProps> = ({
       startAdornment={
         <Icon path={mdiAccountSearch} title="Search contacts" size="24px" />
       }
-      value={searchTerm}
       placeholder={placeholder ?? t('Search')}
       onChange={(event) => {
         onChange(event.target.value);
```

Comment:
@reldredge71 @OzzieOrca  Do you think we should debounce this ```onChange```? I could just see someone typing in a last name like ```Huffman``` and it refetching for each letter. With a debounce, we could limit the number of calls made.

Reply:
@TheNoodleMoose yes that's a good idea.  I'm pretty sure what I have now will make an API call with each character added/taken away.

Reply:
Ya that's a great idea. This is an expensive query to load a bunch of contacts every time a character changes.

https://github.com/nkbt/react-debounce-input looks kinda nice after having scrolled through https://stackoverflow.com/questions/23123138/perform-debounce-in-react-js. A lot of ideas there could work but they all seem to have a bunch of code to add to each component. A lot of them seem to depend on lodash.debounce which is fine.

I didn't research this very long, maybe there are better ideas out there. I didn't see an easy way to debounce this on the Apollo side.